### ISSUE TITLE
Unable to use config file from in current working directory or with explicit extension using CLI

### ISSUE THREAD

**Issue by mcous** [Mermade/openapi-codegen#131]
This is a very cool tool; thanks for your work on it! I've been playing around with it for a few minutes, and I ran into the following issue.

## overview

I would like to use a custom config file with the CLI to generate my client, as none of the existing configs match my needs. The first named parameter of the CLI allows me to pass in a path to a config file, but it seems to arbitrarily prevent me from using any file in the current working directory.

## expected behavior

Based on the README.md + CLI usage instructions, I would expect the following to work:

```shell
# specify a file in the cwd
cg ./custom-config ./openapi.json

# specify an absolute path to a file with its extension
cg /Users/user/project/custom-config.json ./openapi.json
```

## actual behavior

```shell
# cg --verbose ./custom-config openapi.json
Error: ENOENT: no such file or directory, open '/Users/user/project/node_modules/openapi-codegen/configs/custom-config.json'

# cg --verbose /Users/user/project/custom-config.json openapi.json
Error: ENOENT: no such file or directory, open '/Users/user/project/custom-config.json.json'
```

## notes + proposal

The code that reads the config file from the argument makes two errors, as I can see it:

https://github.com/Mermade/openapi-codegen/blob/2cfc04c512db0f52d60cb293ef66f99a3233e724/cg.js#L65-L71

1. If the result of `path.dirname` on the config file arg is `'.'`, assume the user was trying to load something from the project's `configs` folder
    - This behavior was introduced in 2ee839466dd788d30d1762bfa50b655c4cd43e41, and I can't see anything from that commit to explain why that logic was introduced
2. `.json` is always appended to the path string

### proposal

I'm happy to open a PR to fix these issues! I think the following would be the appropriate changes:

- Remove `(configPath === '.')` check
    - If `path.dirname` returns a directory, the CLI should assume the user has provided a file path relative to the cwd
- Only add `.json` if `path.extname` on the basename returns an empty string
    - From reading the code, this may have the happy accident of allowing the user to specify a custom config as a `.yml` file

**Comment by MikeRalphson** [comment]
Thank you for the diagnosis. I'd be happy to see your PR.

**Comment by MikeRalphson** [comment]
Should be fixed in `openapi-codegen@2.2.1`.
